### ISSUE TITLE
Task list for Week of 9/12

### ISSUE THREAD

**Issue by kdahlquist** [dondi/GRNsight#329]
Each week this semester, we will create an issue with a task list for that week.  I'm starting one for our next meeting on 9/12.

- [x] @anuvarsh and @yshin4 will meet for code walkthrough on Monday at 1:00 #265.
- [x] @yshin4 will add herself and make needed changes to the People page #328 
- [x] @yshin4 needs to read and agree to code of conduct #325 
- [ ] Issue #332, merging branches needs to be completed before diving into this week's work
- [ ] @anuvarsh is working on showing/hiding all weights #285 
- [x] @NAnguiano is working on refactoring code for the latest version of node-xlsx #204 
    * this is basically done and can be closed once the merging of #332 is complete
- [ ] @mihirsamdarshi and @yshin4 will work on updating the state diagram #326  
- [ ] @mihirsamdarshi and @yshin4 will then work on the test refactoring #300 
- [x] @NAnguiano and @anuvarsh are going to figure out what packages are not being loaded through `npm install GRNsight` and write up an issue for it.
- [ ] @NAnguiano will move on to investigating #159 (bounding box size) and #294 (signal node not responding to physics)
- [x] everyone, put up flyers to advertise GRNsight seminar and party on friday
- [x] advertise GRNsight publication on social media

**Comment by kdahlquist** [comment]
This is a reminder to everyone to post updates as a comment on whatever issues you are working on.  I've updated the task list on this issue, so please take a look at it.

**Comment by kdahlquist** [comment]
Migrated open tasks to new issue #336, so closing this.
